contract/test/test-contract.js

```diff
@@ -22,19 +23,33 @@ import { startOfferUpContract } from '../src/offer-up-proposal.js';
 const myRequire = createRequire(import.meta.url);
 const contractPath = myRequire.resolve(`../src/offer-up.contract.js`);
 
-/** @type {import('ava').TestFn<Awaited<ReturnType<makeTestContext>>>} */
-const test = anyTest;
```

Comment:
why change this?

Reply:
sorry, wasn't sure what this was achieving but I now finally understand. reverting while addressing comments.

Reply:
reminds me of https://thoughtbot.com/blog/chestertons-fence =)